### DISCUSSION TITLE
How can i apply workflow on modification any record?

### DISCUSSION THREAD

**Discussion by Turkialessa** [directus/directus#7422] [Archive]
How can I apply workflow on modification of any record?

I mean by that, if I have one of the records that are public, and it has been modified, it is not available to people except after approval by the admin.

**Comment by rijkvanzanten** [comment]
You could setup a custom hook (https://docs.directus.io/guides/api-hooks/) to automatically update a field to a different value when it's updated (like setting `status` to `under-review` on create/update). Then using permissions, you can only allow update access to items where that status field isn't under-review
